### DISCUSSION TITLE
Sorting capability

### DISCUSSION THREAD

**Discussion by ChefDaddyJay** [jdwilkin4/fcc-forum-clone#50] [Ideas]
Once post metrics are displayed properly, it makes sense to be able to sort the posts by those metrics. Topics we might discuss:

- What sorting algorithm do we want to use? We're only talking about 30 posts here, so realistically there won't be much difference, I imagine. But, for sake of learning and discussion, it's an interesting topic.
- Things we might sort by include:

- [ ] Replies
- [ ] Views
- [ ] Activity
- [ ] Group by category?
- [ ] Number of contributors?
- [ ] Any others?

**Comment by jdwilkin4** [comment]
We might be limited by the API's capabilities.
It looks like we could sort by replies, views and activity.
